src/extension/background-script/router.js

```diff
@@ -30,6 +30,7 @@ const routes = {
   lnurl,
   lnurlPay,
   sendPayment: ln.sendPayment,
+  weblnPay: ln.weblnPay,
```

Comment:
maybe that's a good time not to use those namespaces that we actually support? 

`webln.sendPayment` (is what the content script calls) vs. `ln.sendPayment` is what we internally call to actually perform the payment.  - though it might not be good to have to files with the same name? (but different folder?)

right now I would reverse the names: 
weblnsendPayment for what the content script calls (sendPayment right now) and then `sendPayment` to do the actual payment (your weblnPay) - I see anything webln close to the website...